bedrock/mozorg/templates/mozorg/contribute/signup.html

```diff
@@ -62,6 +45,16 @@ <h1 class="page-title">{{ _('To get started, tell us what interests you.') }}</h
             </li>
           {% endfor %}
         </ul>
+
+        {% if form.errors %}
+          {{ form.non_field_errors()|safe }}
+          <ul class="errorlist indented">
```

Comment:
Bah, sorry for not being clear - I mean the way it is displayed in the UI. Seems a bit disjointed from the grid?